### PR TITLE
Rollout Llama as default model to 75% of users

### PR DESCRIPTION
**Pull request by nvonpentz**
Follows on https://github.com/brave/brave-variations/pull/1249

### CODE REVIEW

**Review by LorenzoMinto — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
Uplift into `main/production` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
